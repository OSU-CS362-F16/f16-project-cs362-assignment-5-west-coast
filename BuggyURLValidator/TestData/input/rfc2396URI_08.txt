file://path/index.html
file:/path/index.html
file:///C:/cygwin64/home/Corinna/cs362/project/BuggyURLValidator/target/site/cobertura/index.html
http://localhost
file:index.html
