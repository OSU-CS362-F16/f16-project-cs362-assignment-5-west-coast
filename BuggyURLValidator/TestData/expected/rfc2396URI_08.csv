testcase,scheme,ALLOW_LOCAL_URLS,NO_FRAGMENTS,ALLOW_2_SLASHES,ALLOW_ALL_SCHEMES,regex,file://path/index.html,file:/path/index.html,file:///C:/cygwin64/home/Corinna/cs362/project/BuggyURLValidator/target/site/cobertura/index.html,http://localhost,file:index.html
0,{[http][https]},OFF,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
1,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
2,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
3,{[http][https][ftp]},OFF,OFF,OFF,ON,DEFAULT,false,false,true,false,false
4,{[http][https][ftp]},OFF,OFF,ON,OFF,DEFAULT,false,false,false,false,false
5,{[http][https][ftp]},OFF,OFF,ON,ON,DEFAULT,false,false,true,false,false
6,{[http][https][ftp]},OFF,ON,OFF,OFF,DEFAULT,false,false,false,false,false
7,{[http][https][ftp]},OFF,ON,OFF,ON,DEFAULT,false,false,true,false,false
8,{[http][https][ftp]},OFF,ON,ON,OFF,DEFAULT,false,false,false,false,false
9,{[http][https][ftp]},OFF,ON,ON,ON,DEFAULT,false,false,true,false,false
10,{[http][https][ftp]},ON,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
11,{[http][https][ftp]},ON,OFF,OFF,ON,DEFAULT,false,false,true,false,false
12,{[http][https][ftp]},ON,OFF,ON,OFF,DEFAULT,false,false,false,false,false
13,{[http][https][ftp]},ON,OFF,ON,ON,DEFAULT,false,false,true,false,false
14,{[http][https][ftp]},ON,ON,OFF,OFF,DEFAULT,false,false,false,false,false
15,{[http][https][ftp]},ON,ON,OFF,ON,DEFAULT,false,false,true,false,false
16,{[http][https][ftp]},ON,ON,ON,OFF,DEFAULT,false,false,false,false,false
17,{[http][https][ftp]},ON,ON,ON,ON,DEFAULT,false,false,true,false,false
18,{[http][https]},OFF,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
19,{[http][https]},OFF,OFF,OFF,ON,DEFAULT,false,false,true,false,false
20,{[http][https]},OFF,OFF,ON,OFF,DEFAULT,false,false,false,false,false
21,{[http][https]},OFF,OFF,ON,ON,DEFAULT,false,false,true,false,false
22,{[http][https]},OFF,ON,OFF,OFF,DEFAULT,false,false,false,false,false
23,{[http][https]},OFF,ON,OFF,ON,DEFAULT,false,false,true,false,false
24,{[http][https]},OFF,ON,ON,OFF,DEFAULT,false,false,false,false,false
25,{[http][https]},OFF,ON,ON,ON,DEFAULT,false,false,true,false,false
26,{[http][https]},ON,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
27,{[http][https]},ON,OFF,OFF,ON,DEFAULT,false,false,true,false,false
28,{[http][https]},ON,OFF,ON,OFF,DEFAULT,false,false,false,false,false
29,{[http][https]},ON,OFF,ON,ON,DEFAULT,false,false,true,false,false
30,{[http][https]},ON,ON,OFF,OFF,DEFAULT,false,false,false,false,false
31,{[http][https]},ON,ON,OFF,ON,DEFAULT,false,false,true,false,false
32,{[http][https]},ON,ON,ON,OFF,DEFAULT,false,false,false,false,false
33,{[http][https]},ON,ON,ON,ON,DEFAULT,false,false,true,false,false
34,{[http][https][ftp]},OFF,OFF,OFF,OFF,^[\w-\\.]*$,false,false,false,true,false
35,{[http][https][ftp]},OFF,OFF,OFF,ON,^[\w-\\.]*$,true,false,true,true,false
36,{[http][https][ftp]},OFF,OFF,ON,OFF,^[\w-\\.]*$,false,false,false,true,false
37,{[http][https][ftp]},OFF,OFF,ON,ON,^[\w-\\.]*$,true,false,true,true,false
38,{[http][https][ftp]},OFF,ON,OFF,OFF,^[\w-\\.]*$,false,false,false,true,false
39,{[http][https][ftp]},OFF,ON,OFF,ON,^[\w-\\.]*$,true,false,true,true,false
40,{[http][https][ftp]},OFF,ON,ON,OFF,^[\w-\\.]*$,false,false,false,true,false
41,{[http][https][ftp]},OFF,ON,ON,ON,^[\w-\\.]*$,true,false,true,true,false
42,{[http][https][ftp]},ON,OFF,OFF,OFF,^[\w-\\.]*$,false,false,false,true,false
43,{[http][https][ftp]},ON,OFF,OFF,ON,^[\w-\\.]*$,true,false,true,true,false
44,{[http][https][ftp]},ON,OFF,ON,OFF,^[\w-\\.]*$,false,false,false,true,false
45,{[http][https][ftp]},ON,OFF,ON,ON,^[\w-\\.]*$,true,false,true,true,false
46,{[http][https][ftp]},ON,ON,OFF,OFF,^[\w-\\.]*$,false,false,false,true,false
47,{[http][https][ftp]},ON,ON,OFF,ON,^[\w-\\.]*$,true,false,true,true,false
48,{[http][https][ftp]},ON,ON,ON,OFF,^[\w-\\.]*$,false,false,false,true,false
49,{[http][https][ftp]},ON,ON,ON,ON,^[\w-\\.]*$,true,false,true,true,false
50,{[http][https]},OFF,OFF,OFF,OFF,^[\w-\\.]*$,false,false,false,true,false
51,{[http][https]},OFF,OFF,OFF,ON,^[\w-\\.]*$,true,false,true,true,false
52,{[http][https]},OFF,OFF,ON,OFF,^[\w-\\.]*$,false,false,false,true,false
53,{[http][https]},OFF,OFF,ON,ON,^[\w-\\.]*$,true,false,true,true,false
54,{[http][https]},OFF,ON,OFF,OFF,^[\w-\\.]*$,false,false,false,true,false
55,{[http][https]},OFF,ON,OFF,ON,^[\w-\\.]*$,true,false,true,true,false
56,{[http][https]},OFF,ON,ON,OFF,^[\w-\\.]*$,false,false,false,true,false
57,{[http][https]},OFF,ON,ON,ON,^[\w-\\.]*$,true,false,true,true,false
58,{[http][https]},ON,OFF,OFF,OFF,^[\w-\\.]*$,false,false,false,true,false
59,{[http][https]},ON,OFF,OFF,ON,^[\w-\\.]*$,true,false,true,true,false
60,{[http][https]},ON,OFF,ON,OFF,^[\w-\\.]*$,false,false,false,true,false
61,{[http][https]},ON,OFF,ON,ON,^[\w-\\.]*$,true,false,true,true,false
62,{[http][https]},ON,ON,OFF,OFF,^[\w-\\.]*$,false,false,false,true,false
63,{[http][https]},ON,ON,OFF,ON,^[\w-\\.]*$,true,false,true,true,false
64,{[http][https]},ON,ON,ON,OFF,^[\w-\\.]*$,false,false,false,true,false
65,{[http][https]},ON,ON,ON,ON,^[\w-\\.]*$,true,false,true,true,false
66,{[http][https][ftp]},OFF,OFF,OFF,OFF,DEFAULT,false,false,false,false,false
